pkg/ui/react-app/src/thanos/pages/notFound/NotFound.tsx

```diff
@@ -0,0 +1,9 @@
+import React, { FC } from 'react';
+
+interface NotFoundProps {
+  default: boolean;
+}
+
+const NotFound: FC<NotFoundProps> = () => <h3>404, Page Not Found</h3>;
```

Comment:
It'll be better to make this a bit more clean/visually pleasing. We have some container components in `reactstrap` (like `Contatiner` etc.). Try doing something like
```jsx
<Container fluid>
  <h3>404, Page Not Found</h3>
</Container>
```
to get the `<h3>` centered. Maybe we can also add a link here saying `"Return to homepage"` or simply `"Go back"`.